### ISSUE TITLE
[Bug]: 25.0.0 Docker images being generated for armv7 despite architecture deprecation

### ISSUE THREAD

**Issue by sunjam** [nextcloud/docker#1852]

**Comment by martadinata666** [comment]
may I get references? as I'm on 32bit,and don't get any notifications? 🤔

**Comment by sunjam** [comment]
There are no notifications.  This is being found through testing and by reviewing pull requests @martadinata666.

- https://github.com/nextcloud/server/pull/34905

`25.0.2` will resume offering 32-bit support on eta December 1st _until nc25 reaches EOL end of 2022_.
- Until then 32-bit people can stay on `24.*`
- [See forum post here](https://help.nextcloud.com/t/32-bit-support-returns-estimate-dec-1st-with-25-0-2-but-officially-ends-as-nc25-reaches-eol-in-about-a-year/149032)

**Comment by sunjam** [comment]
32bit should be supported in 25.0.2 on eta December 1st.

25.0.0 and 25.0.1 are not supported

**Comment by sunjam** [comment]
> So that means it is still supported.

Deprecation is still happening.  25.0.0 is not supported and neither is 25.0.1 so you can remove those images.  It will be supported again in December 25.0.2 until the release of 26 cutoff.  You'll still still need to plan for removing these images alongside 26.  @szaimen might be able to advise you.

**Comment by szaimen** [comment]
Yes indeed. Nextcloud 26 will remove 32-bit compatibility officially.

**Comment by sunjam** [comment]
@szaimen currently there are 32-bit docker images being generated for [official Nextcloud Docker Hub](https://hub.docker.com/_/nextcloud) and people are of course unable to use them and getting confused.  Wondering if you could help advise on any way to prevent these broken images from staying available.
25.0.0
25.0.1
and of course 26+

Thanks!

**Comment by szaimen** [comment]
Sorry, I don't have control over the "official" docker containers. For that you need to ask @J0WI. However indeed for 26 should 32-bit images not get build anymore.

**Comment by szaimen** [comment]
They are supposed to switch to 64-bit.

**Comment by sunjam** [comment]
To be fair 32-bit has been in deprecation for years and they don't have to
support our old devices moving forward at all, but yes to better
documentation and communication.

**Comment by LeSpocky** [comment]
> They are supposed to switch to 64-bit.

What will actually happen is rather the following: people won't throw away their RPi, Odroid, Cubie, or whatever armv7 single board computer they use, but simply run it with NC 24 or 25 forever. Exposing those eventually old and unsupported software versions to the internet.

**Comment by szaimen** [comment]
https://help.nextcloud.com/t/deprecation-of-32-bit-support-ends-completely-with-nextcloud-26/150926
